runtime/vcache/shadow.go

```diff
@@ -141,6 +150,8 @@ func newShadow(m vng.Metadata, n *vng.Nulls, nullsCnt uint32) shadow {
 		}
 		nullsCnt += m.Count
 		return newShadow(m.Values, m, nullsCnt)
+	case *vng.Error:
+		return &error_{vals: newShadow(m.Values, n, nullsCnt), nulls: nulls{meta: n}}
```

Comment:
Nit: Follow the *vng.Named case.
```suggestion
		return &error_{newShadow(m.Values, n, nullsCnt), nulls{meta: n}}
```